lib/checkForNewCodeowner.js

```diff
@@ -64,7 +66,9 @@ const getNewCodeOwners = (file) => {
 const getNewCodeOwnerFiles = (file) => {
   const changesArray = file.patch.split(newLine);
   const newAdditions = changesArray.filter((change) => {
-    return change.startsWith(newAddition);
+    return (
+      change.startsWith(newAddition)&& !change.startsWith(newCommentAddition)
+    );
```

Comment:
We have this twice in this PR, should we separate it into its own function and use it in both places?

Reply:
Created the function!